src/main/java/com/google/sps/servlets/CredentialVerifierServlet.java

```diff
@@ -20,13 +20,13 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
     AuthorizationCodeFlow flow;
     try {
       flow = Utils.newFlow();
-
+      response.setContentType("text/html");
       User user = UserServiceFactory.getUserService().getCurrentUser();
       if (user == null) {
         response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+        response.getWriter().print("false");
```

Comment:
Since the Http status is already pointing to an error, not sure whether you should write something to the response here.